contracts/SpokePool.sol

```diff
@@ -1065,10 +1098,99 @@ abstract contract SpokePool is
         return block.timestamp; // solhint-disable-line not-rely-on-time
     }
 
+    /**
+     * @notice Returns the deposit ID for an unsafe deposit. This function is used to compute the deposit ID
+     * in unsafeDepositV3 and is provided as a convenience.
+     * @dev msg.sender and depositor are both used as inputs to allow passthrough depositors to create unique
+     * deposit hash spaces for unique depositors.
+     * @param sender The msg.sender address used as input to produce the deposit ID.
```

Comment:
This wording is a bit counter-intuitive for me because it's relying on the implementation details of another function (`unsafeDepositV3()`). i.e. it's confusing that `msg.sender` is referenced in the comment, but `sender` is an input argument and can be any address.

Is `relayer` a candidate variable name? It implies that the relayer's address is needed to produce the deterministic deposit ID. In context of a relayer filling via a contract it would be the contract address, not the EOA, so idk whether that introduces a possible additional source of confusion.

Reply:
I think msgSender maybe is a better variable name

Reply:
Because maybe an aggregator also wants to use the unsafeDeposit method but there would be no natural "relayer" to fill in, right? Maybe i'm over thinking it

Reply:
https://github.com/across-protocol/contracts/pull/583/commits/78eb4c03d1954bc2515cbee25f07733d2c1aa1b3